### PR TITLE
Adding acorn version as command (#1569)

### PR DESCRIPTION
**Pull request by imrajdas**
### Checklist
- [x] The title of this PR would make a good line in Acorn's Release Note's Changelog
- [x] The title of this PR ends with a link to the main issue being address in parentheses, like: `This is a title (#1216)`. [Here's an example](https://github.com/acorn-io/acorn/pull/1199)
- [x] All relevant issues are referenced in the PR description. *NOTE: don't use [GitHub keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) that auto-close issues*
- [x] Commits follow [contributing guidance](https://github.com/acorn-io/acorn/blob/main/CONTRIBUTING.md#commits)
- [ ] Automated tests added to cover the changes. If tests couldn't be added, an explanation is provided in the Verification and Testing section
- [x] Changes to user-facing functionality, API, CLI, and upgrade impacts are clearly called out in PR description
- [x] PR has at least two approvals before merging (or a reasonable exception, like it's just a docs change)

### CODE REVIEW

**Comment by imrajdas** [comment]
Issue- https://github.com/acorn-io/acorn/issues/1569

**Comment by tylerslaton** [comment]
@imrajdas Thank you very much for the contribution! CI is red because some docs were not generated and committed. Could you please run `make validate` locally and commit the result?

File: pkg/cli/version.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	"fmt"
+
+	"github.com/acorn-io/acorn/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func NewVersion(c CommandContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "version",
+		Short:   "version for acorn",
```
Comment (njhale) [inline pkg/cli/version.go:13]:
```suggestion
		Short:   "Version information for acorn",
```

**Review by njhale — COMMENTED** [review]
Looking great!

Just one suggestion while we're here.

File: pkg/cli/version.go (hunk 1 above)
Comment (imrajdas) [reply pkg/cli/version.go]: Done

**Comment by imrajdas** [comment]
@tylerslaton @njhale - all checks are passed now.

File: pkg/cli/version.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+func NewVersion(c CommandContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "version",
+		Short:   "Version information for acorn",
+		Example: "acorn version",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Printf("acorn version %s\n", version.Get().String())
+		},
+		ValidArgsFunction: newCompletion(c.ClientFactory, appsCompletion).withShouldCompleteOptions(onlyNumArgs(1)).complete,
+	}
```
Comment (njhale) [inline pkg/cli/version.go:19]: didn't notice this the first time, but it looks like this is adding completion from the `apps` command to `acorn version`.

@imrajdas Can you change this to accept no args?

```suggestion
	cmd := &cobra.Command{
		Use:     "version",
		Short:   "Version information for acorn",
		Example: "acorn version",
		Run: func(cmd *cobra.Command, args []string) {
			fmt.Printf("acorn version %s\n", version.Get().String())
		},
		Args: cobra.NoArgs,
	}
```

Should be good to merge afterwards.

**Review by njhale — CHANGES_REQUESTED** [review]
@imrajdas I missed something in my earlier review. Should be a quick change.

File: pkg/cli/version.go (hunk 2 above)
Comment (imrajdas) [reply pkg/cli/version.go]: done

**Review by tylerslaton — APPROVED** [review]

**Pull request merged**